rdf_differ/skos_history_wrapper.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
We are missing the functionality here where the bash script is executed.

Reply:
I'll add it when all is done, as it's just a single line of code:
```python
subprocess.run(["./load_versions.sh -f {dataset}.config"])
```